### PR TITLE
update dev versions

### PR DESCRIPTION
**Pull request by avpeery**
Next release branch is cut ([release-v2.8.0](https://github.com/mapbox/mapbox-gl-js/tree/release-v2.8.0)), updating dev versions. We can cherry-pick the codebuild set-up when it's ready. 

## Launch Checklist

 - [x] briefly describe the changes in this PR
 - [x] apply changelog label ('bug', 'feature', 'docs', etc) or use the label 'skip changelog'

### CODE REVIEW

File: src/style-spec/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mapbox/mapbox-gl-style-spec",
   "description": "a specification for mapbox gl styles",
-  "version": "13.24.0-dev",
+  "version": "13.24.1-dev",
```
Comment (ryanhamley) [inline src/style-spec/package.json:4]: Should this be `13.25.0`?

Comment (avpeery) [reply src/style-spec/package.json:4]: Yes

**Review by ryanhamley — APPROVED** [review]

**Comment by avpeery** [comment]
Waiting on release blockers: https://github.com/mapbox/mapbox-gl-js/issues/11587

**Review by SnailBones — APPROVED** [review]

**Review by SnailBones — APPROVED** [review]
👍

**Pull request merged**